_includes/_code/lakes-interactive/lib/1st-try.dart

```diff
@@ -0,0 +1,138 @@
+import 'package:flutter/material.dart';
+// Uncomment lines 3 and 6 to view the visual layout at runtime.
+//import 'package:flutter/rendering.dart' show debugPaintSizeEnabled;
+
+void main() {
+  //debugPaintSizeEnabled = true;
+  runApp(new MyApp());
+}
+
+IconData icon = Icons.star;
+
+class FavoriteWidget extends StatefulWidget {
+  /**
+  new Icon(
+  icon,
+  color: Colors.red[500],
+  ),
+  new Text('41'),
+      */
+  
+}
+
+
+//class MyApp extends StatefulWidget {
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    Widget titleSection = new Container(
+      padding: const EdgeInsets.all(32.0),
+      child: new Row(
+        children: [
+          new Expanded(
+            child: new Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                new Container(
+                  padding: const EdgeInsets.only(bottom: 8.0),
+                  child: new Text(
+                    'Oeschinen Lake Campground',
+                    style: new TextStyle(
+                      fontWeight: FontWeight.bold,
+                    ),
+                  ),
+                ),
+                new Text(
+                  'Kandersteg, Switzerland',
+                  style: new TextStyle(
+                    color: Colors.grey[500],
+                  ),
+                ),
+              ],
+            ),
+          ),
+          new FavoriteWidget(),
+        ],
+      ),
+    ); // end of "Widget titleSection"
+
+    Column buildButtonColumn(IconData icon, String label) {
+      List<Widget> widgets = new List(2);
+      Color color = Theme.of(context).primaryColor;
+
+      widgets[0] = new Icon(icon, color: color);
+      widgets[1] = new Container(
```

Comment:
Rather than creating this separate List, I'd just put these widgets inside a list literal on line 71 i.e.,

```
children: [
  new Icon(...),
  new Container(...),
]
```